include/SugarCache/SugarCache.php

```diff
@@ -165,8 +167,9 @@ public static function cleanFile($file)
         // Zend OPcache
         if (
             extension_loaded('Zend OPcache') &&
+            ((function_exists('opcache_get_status') &&
             ($opcache_status = opcache_get_status(false)) !== false &&
-            $opcache_status['opcache_enabled']
+            $opcache_status['opcache_enabled'] ) || ini_get('opcache.enable') )
```

Comment:
Hi @JanSiero,

Same here, could we do something like the following please?

```php
$opcacheLoaded = extension_loaded('Zend OPcache');
$opcache_status_enabled = false;

if(function_exists('opcache_get_status')) {
    $opcache_status = opcache_get_status(false);
    $opcache_status_enabled = $opcache_status !== false && ($opcache_status['opcache_enabled'] ?? false);
}

$ini_opcache_enabled = ini_get('opcache.enable');


if (
    $opcacheLoaded &&
    ($opcache_status_enabled || $ini_opcache_enabled)
) {
    // three attempts incase concurrent opcache operations pose a lock
    for ($i = 3; $i && !opcache_invalidate($file, true); --$i) {

```

Reply:
Idem

```php
Hi @clemente-raposo your code always retrieves the opcache status by ini_get, even when opcache_get_status was successful, although it was meant as a fallback.

To reduce the nr. of variables, and improve readability I'd propose this code. Can you assess this? Would you like me to adapt my pull request accordingly?

```php
        // Zend OPcache
        if (extension_loaded('Zend OPcache') {
            $opcache_status_enabled = false;
            
            if(function_exists('opcache_get_status')) {
                $opcache_status = opcache_get_status(false);
                $opcache_status_enabled = $opcache_status !== false && ($opcache_status['opcache_enabled'] ?? false);
            } else {
                $opcache_status_enabled = ini_get('opcache.enable');
            }
            
            if ($opcache_status_enabled) {
                // three attempts incase concurrent opcache operations pose a lock
                for ($i = 3; $i && !opcache_invalidate($file, true); --$i) {
                    sleep(0.2);
                }
            }
        }
```

Reply:
@JanSiero thank you for the changes